src/coreclr/src/vm/clrex.h

```diff
@@ -30,6 +30,9 @@ struct StackTraceElement
     // TRUE if this element represents the last frame of the foreign
     // exception stack trace.
     BOOL			fIsLastFrameFromForeignStackTrace;
+    // TRUE if the ip needs to be adjusted back to the calling or
+    // throwing instruction.
+    BOOL            fAdjustOffset;
```

Comment:
This type is separately defined in SOS [here](https://github.com/dotnet/diagnostics/blob/master/src/SOS/Strike/strike.cpp#L2245). Data is extracted as a binary format [here](https://github.com/dotnet/diagnostics/blob/master/src/SOS/Strike/strike.cpp#L2669) and then parsed using this type definition [here](https://github.com/dotnet/diagnostics/blob/master/src/SOS/Strike/strike.cpp#L2420). I'd expect this breaks !PrintException in SOS.

Reply:
I didn't see this SOS dependency and I was looking at this code for the SOS line number fix.  Thanks for catching this.